### ISSUE TITLE
A.02.IR02.IR03.at.least.one.recource.locator

### ISSUE THREAD

**Issue by jensscheerlinck** [inspire-eu-validation/ats-invocable-sds#6]
_This issue has been extracted from the issue list on:https://ies-svn.jrc.ec.europa.eu/issues/2685_
# Comment

The test method is should mention that only the gmd:CI_OnlineResource with a specific description are considered.
In addition, the XPath uses the gmd:function element while the TG prescribes gmd:description

**Comment by cportele** [comment]
These SDS-related conformance classes / ATSs have not been part of the review: https://ies-svn.jrc.ec.europa.eu/issues/2685

Proposed resolution:
- Keep the issue open and revisit as part of the future work on the SDS ATS.

**Comment by cportele** [comment]
Changed to rejected and wontfix as this reflects the status during this review. The issue may be revisited once the SDS ATS is ready for review.
